src/utils/utils.js

```diff
@@ -11,18 +11,143 @@ const disabilityMap = {
   AUTYZM: "z autyzmem, w tym z zespołem Aspergera"
 };
 
-const calculateValues = values => {
-  const reason = values.applicant.reason;
-  let secondReason = values.applicant.secondReason;
-  if (reason && secondReason) {
-    const firstReason = disabilityMap[reason];
-    secondReason = disabilityMap[secondReason];
-    values.applicant.multipleDisability = [firstReason, secondReason];
-    values.applicant.reason = "SPRZEZONA";
-  }
-  return values;
-};
+class DocumentData {
+  constructor(values) {
+    this.values = values;
+  }
+
+  isMultipleDisability() {
+    return Boolean(
+      this.values.applicant.reason && this.values.applicant.secondReason
+    );
```

Comment:
We should not use Boolean constructor, could you change it to:
`return !!(this.values.applicant.reason && this.values.applicant.secondReason)`